custom_metrics/privacy.js

```diff
@@ -1,24 +1,312 @@
-    var pattern = new RegExp(words, 'i');
+    let pattern = new RegExp('\\b(?:' + words + ')\\b', 'i');
 
-    return Array.from(document.querySelectorAll('a')).filter((a) =>
-      a.innerText.match(pattern)
-    ).length;
+    let privacy_links = Array.from(document.querySelectorAll('a')).filter(a => a.innerText.match(pattern));
+
+    return privacy_links.map(link => link.innerText);
```

Comment:
Returning the whole text may make aggregation difficult later, for example accounting for additional text / whitespace / case-sensitivity. WDYT about mapping links to the matching word pattern specifically?

Reply:
Or both? Having the whole text might be more useful for debugging, the specific pattern more for additional analysis.

Reply:
Stored both in 00f7160. Feel free to revert, but in that case we could also consider using [`test`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test) instead of `match`.
WPT run (rtl.de): https://www.webpagetest.org/custom_metrics.php?test=210630_AiDcD7_c350b1535df7dec7bddbafff01698b79&run=1&cached=0